src/Pim/Bundle/CatalogBundle/Elasticsearch/ProductIndexer.php

```diff
@@ -16,7 +18,7 @@
  * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
  * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
  */
-class ProductIndexer implements IndexerInterface, BulkIndexerInterface
+class ProductIndexer implements IndexerInterface, BulkIndexerInterface, RemoverInterface, BulkRemoverInterface
```

Comment:
Shouldn't we create a `ProductIndexerInterface` that implements those four interfaces, and only implement this one in `ProductIndexer`?